inference-engine/thirdparty/clDNN/src/include/kernel_selector_helper.h

```diff
@@ -23,6 +23,8 @@
 
 using namespace cldnn;
 
+static std::mutex my_test;
```

Comment:
seems not being used

Reply:
Yes, I know, It's remained code for debugging. I forget to remove it. I will remove it. thanks.

Reply:
Remove it. thanks.